apps/src/netsim/netsim.js

```diff
@@ -125,7 +133,7 @@ NetSim.prototype.init = function(config) {
    * Configuration for the loaded level
    * @type {netsimLevelConfiguration}
    */
-  this.level = config.level;
+  this.level = netsimUtils.scrubLevelConfiguration_(config.level);
```

Comment:
You might be able to do this by providing JSON.parse a reviver method https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/parse. I seem to remember trying to play with revivers before and it being more difficult than I expected however.